attesters/hydra-s1/index.ts

```diff
@@ -1,11 +1,17 @@
-import { AttestationsCollection } from "../../src/topics/attestations-collection/attestations-collection";
-import { Badge } from "../../src/topics/badge";
+import { AttestationsCollection } from "../../src/topics/attestations-collection";
 import { Attester } from "../../src/topics/attester";
+import { Badge } from "../../src/topics/badge";
 import { Group } from "../../src/topics/group";
 
-export const hydraS1SimpleAttester = async () =>
+export default async () =>
   new Attester({
-    collectionIdFirst: 10000001,
+    name: "hydra-s1-simple",
+    configurations: {
+      polygon: {
+        firstCollectionId: 10000001,
+        address: "",
+      },
+    },
     attestationsCollections: [
       // Sismo friends
       new AttestationsCollection({
```

Comment:
I think we could just change the groups declaration here to 
groupFetcher: () => [await Group.store.latest("sismo-diggers")] 
And it removes all the pb with groups. 
Also without doing this, it will download all groups for just the badge API, that don't seems to be logic